docs/sdks/sdk-ref-cplusplus.md

```diff
@@ -23,13 +23,17 @@ This SDK simplifies integration with UID2 for any DSPs or UID2 sharers who are u
 | :--- | :--- | :--- | :--- | :--- | :--- |
 | &#9989; | &#9989; | &#8212; | &#8212; | &#8212; | &#8212; |
 
-## API Permissions
+## UID2 Account Setup
+
+To integrate with UID2, you'll need to have a UID2 account. If you haven't yet created an account, first follow the steps described on the [Account Setup](../getting-started/gs-account-setup.md) page.
 
-To use this SDK, you'll need to complete the UID2 account setup by following the steps described in the [Account Setup](../getting-started/gs-account-setup.md) page.
+## API Permissions
```

Comment:
again don't need separate API Permission section any more - applies to all 4 SDK pages